app/controllers/users.js

```diff
@@ -1,99 +1,90 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
+const mongoose = require('mongoose'),
+
   User = mongoose.model('User');
-var avatars = require('./avatars').all();
+const avatars = require('./avatars').all();
+const jwt = require('jsonwebtoken');
 
-/**
- * Auth callback
- */
-exports.authCallback = function(req, res, next) {
+const secret = 'secretoverhere';
```

Comment:
The secret key is supposed to be secret, you might consider using environment variable